extensions/components/com_redcore/admin/models/translation.php

```diff
@@ -141,6 +141,14 @@ public function save($data)
 		$original = JFactory::getApplication()->input->get('original', array(), 'array');
 		$id = !empty($data['rctranslations_id']) ? (int) $data['rctranslations_id'] : 0;
 
+		//Check if the form is completely empty, and return an error if it is.
+		$dataFilled = RedcoreHelpersTranslation::validateEmptyTranslationData($translation, $translationTable->primaryKeys);
+		if(!$dataFilled)
+		{
+			$this->setError(JText::_('COM_REDCORE_TRANSLATIONS_SAVE_ERROR_EMPTY'));
```

Comment:
If all translated columns are empty then I would remove that translation record from the DB here (if exists)